config/trainer_config.yaml

```diff
@@ -16,6 +16,7 @@ default:
     sequence_length: 64
     summary_freq: 1000
     use_recurrent: false
+    vis_encode_type: default
```

Comment:
If "default" is really the default, you shouldn't need to put it in the configs.